CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ set(RSLANG_BUILD_TOOLS "<none>" CACHE STRING "Rust tools to build (see build.too
 include(TargetArchitecture)
 include(CircuitCompile)
 
+find_package(Boost REQUIRED COMPONENTS json)
```

Comment:
why only json required?

Reply:
Because the boost version from apt-get already contained all other required boost submodules.
But it's a valid point to explicitly list all the required ones